src/raftstore/store/store.rs

```diff
@@ -632,6 +632,7 @@ impl<T: Transport, C: PdClient> Store<T, C> {
                 if let Err(e) = self.pd_worker.schedule(task) {
                     error!("{} failed to notify pd: {}", peer.tag, e)
                 }
+                self.raft_metrics.leader_miss.inc();
```

Comment:
Can the region id be reported in metrics?

Reply:
I used to think so, but I am worried about too many labels. Any suggestions?

> Do not use labels to store dimensions with high cardinality (many different label values), such as user IDs, email addresses, or other unbounded sets of values.

https://prometheus.io/docs/practices/naming/#labels